content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -1,146 +1,404 @@
+   Rounding is applied *after* the discount and *before* the surcharge.
+
+.. tip::
+   To have prices that end in 9.99, set rounding to `10`, surcharge `-0.01`.
+
+Lastly, specify the minimum amount of margin over the base price in the :guilabel:`Margins` field.
+
+Once all formula-related configurations are complete, Odoo provides an example of the formula in
+a blue block to the right of the configurations.
 
-The default pricelist can be replaced when creating a sales order.
+.. example::
+   Here's how it would look to apply 20% discounts, with prices rounded up to 9.99.
 
-.. tip:: You can change the pricelists sequence by drag & drop in list view.
+   .. image:: pricing/formula-discount-example.png
+      :align: center
+      :alt: Example of a 20% discount with prices rounded to 9.99 in Odoo Sales.
 
-Compute and show discount % to customers
-========================================
+Conditions
+----------
 
-In case of discount, you can show the public price and the computed discount % on printed sales
-orders and in your eCommerce catalog. To do so:
+At the bottom of the :guilabel:`Create Pricelist Rules` pop-up form is the :guilabel:`Conditions`
+section.
 
-* Check *Allow discounts on sales order lines* in :menuselection:`Sales --> Configuration -->
-  Settings --> Quotations & Sales --> Discounts`.
-* Apply the option in the pricelist setup form.
+Here, start by selecting one of the options in the :guilabel:`Apply On` field:
 
-.. image:: pricing/discount_options.png
+- :guilabel:`All Products`: the advanced pricelist rule will be applied to all products.
+- :guilabel:`Product Category`: the advanced pricelist rule will be applied to a specific category
+  of products.
+- :guilabel:`Product`: the advanced pricelist rule will be applied to a specific product.
+- :guilabel:`Product Variant`: the advanced pricelist rule will be applied to a specific product
+  variant.
+
+If any of those options, apart from :guilabel:`All Products`, is selected, a new option-specific
+field appears, in which the specific :guilabel:`Product Category`, :guilabel:`Product`, or
+:guilabel:`Product Variant` must be chosen, depending on which :guilabel:`Apply On` option is
+chosen.
+
+Then, select a minimum quantity to be applied to the advanced pricelist rule in the :guilabel:`Min.
+Quantity` field. And, lastly, select a range of dates for the pricelist item validation in the
+:guilabel:`Validity` field.
+
+Once all configurations are complete, either click :guilabel:`Save & Close` to save the advanced
+pricelist rule, and return to the :guilabel:`Pricelists` page. Or, click :guilabel:`Save & New` to
+immediately create another advanced pricelist rule on a fresh, completely blank :guilabel:`Create
+Pricelist Rules` pop-up form.
+
+.. note::
+   If a rule is set for a particular item, and another one for its category, Odoo takes the rule of
+   the item. Make sure at least one pricelist item covers all your products.
+
+Show discount percentage to customers
```

Comment:
This whole section should be moved into the Pricelists > Configuration tab section of the doc.